lib/map_users_accounts.rb

```diff
@@ -20,9 +20,11 @@ def find_or_create_user(account)
       begin
         user = UserProfile.where(account: account).first
         return user if user.present?
-
-        UserProfile.create!(account: account)
-
+        # TODO: somehow get organization from somewhere
+        UserProfile.create!(
+          account: account,
+          organization: Organization.find_or_create_by(name: 'OpenStax')
```

Comment:
And organization should not necessarily be required for a user.  Also I guess it is possible that one user could be in two organizations (imagine if there are 4 XYZ Corp products and for some reason we set them up as separate organizations, but there's one techy that is a member of all 4 ).  So we might need a many to many join table.